test/semantics/omp-clause-validity01.f90

```diff
@@ -27,14 +27,14 @@
   enddo
   !$omp end parallel
 
-  !ERROR: 'SCHEDULE(STATIC)' not allowed in OMP PARALLEL
+  !ERROR: 'SCHEDULE' clause is not allowed on the PARALLEL directive
```

Comment:
I don't think quotes around the clause name are necessary. Being all-caps makes it stand out and there aren't quotes around PARALLEL.

Reply:
okay!

Reply:
Addressed in commit [cb89744](cb89744746186bcf120d8ca62a791e2778366b9b).